composer.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "exercism/php-test-runner",
+    "type": "project",
+    "license": "AGPL-3.0-only",
```

Comment:
Not sure about the license here

Reply:
this is the standard licence that exercism uses for custom tooling

Reply:
I asked for it [in the forum](https://forum.exercism.org/t/which-license-to-use-for-php-test-runner/12385)